v3/plugins/cdr/code-server/2.1650-vsc1.39.2/meta.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v2
```

Comment:
the 2.x versions of cdr/code-server are obsoleted by version 3.6.2. Why do we want them back?

Reply:
because you did a PR named "switch to quay.io" and I revert this one.
if you wanted to cleanup, then it should have been a separate commit or a separate PR or never been approved/merged

Reply:
yeah I guess the conversation between Eric and I last night before I submitted the PR based on his recommendations wasn't captured in the PR itself.